app/lib/widgets/cards/productCard/shared/product_infos.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flex_ui/flex_ui.dart';
+import 'package:flex_ui/utils/extension.dart';
+import 'package:flex_ui/widgets/cards/productCard/shared/product_notation.dart';
+import 'package:flutter/material.dart';
+
+class ProductInfos extends StatelessWidget {
+  const ProductInfos(
+      {super.key,
+      this.isLandscape = false,
+      required this.productName,
+      this.productReference,
+      required this.price,
+      this.oldPrice,
+      required this.currency,
+      this.notation});
+
+  final String productName;
+  final String? productReference;
+  final double price;
+  final double? oldPrice;
+  final String currency;
+  final int? notation;
+  final bool isLandscape;
+
+  @override
+  Widget build(BuildContext infosContext) {
+    final context = Theme.of(infosContext);
```

Comment:
`final theme = Theme.of(context)` ? Not sure we need to rename to `infosContext`

Reply:
yes it's because I call my variable context, so linter was confused but I'll change that